lib/rmt/config.rb

```diff
@@ -1,6 +1,10 @@
 require 'config'
 require_relative '../rmt'
 
+Config.setup do |config|
+  config.merge_nil_values = false
```

Comment:
The default behavior has changed in the most recent version, `railsconfig` gem now merges `nil` values by default.